@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -178,6 +180,7 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
     private readonly m_geometries: Geometry[] = [];
     private readonly m_textGeometries: TextGeometry[] = [];
     private readonly m_textPathGeometries: TextPathGeometry[] = [];
+    private readonly m_linePathGeometries: PathGeometry[] = [];
```

Comment:
m_linePathGeometries is called "pathGeometries", so I think it should be either "m_linePathGeometries" and "linePathGeometries", or "m_pathGeometries" and "pathGeometries" to be consistent.

Reply:
Done.